client/test/cypress/support/stepDefinitions/homeSteps.js

```diff
@@ -6,7 +6,7 @@ import * as topMenu from '../fragments/topMenu';
 
 Given('an user with a demo project', () => {
   cy.visit(managementProjects.url);
-  demo.reset();
+ // demo.reset();
```

Comment:
Why is this commented ? 
I think the `reset` is there to make sure that if you launch your test 2 times in a row, it behaves the same.